src/Blueman/Console/Command/ConvertCommand.php

```diff
@@ -65,10 +81,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             );
         }
 
+        $tests = array();
+        if('true' === $input->getOption('tests-include')) {
+
+            $testsFile = $filePath . $input->getOption('tests-filename');
+
+            if($testsFileExists = file_exists($testsFile)) {
```

Comment:
What is the variable `testsFileExists` doing here? It's not being used anywhere else. If you just want to check if the test file exists then do `if (file_exists($testsFile) === true) {` instead.